server/src/services/mail/index.js

```diff
@@ -21,7 +21,8 @@ const sendEmail = (data) => {
 
   transporter.sendMail(mailOptions, (error, info) => {
     if (error) {
-      return console.log(error);
+      console.log(error);
+      return;
```

Comment:
It was better I think to immediately return with console error instead of 2 steps